### DISCUSSION TITLE
Regexp to extract Twitter username from the url to create a custom title

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#105] [Q&A] [answered]
I want to create a new title rule for tweets as shown below

`$title=Tweet/username/((*@clip*))
`

Here twitter `username` corresponds to the part of the url between the `https://twitter.com/` and `/status/1576448835680751616?t=6GbMaJRsU1rMObZotBoSJw&s=19` in the example twiter url shown below.

`https://twitter.com/akassarjian/status/1576448835680751616?t=6GbMaJRsU1rMObZotBoSJw&s=19`

What regexp should I use to extract the username part of the url ?

**Comment by buggyj** [comment, marked as answer]
try

```
#rule=/https:\/\/twitter.com\/(.*?)\/status[\s\S]*$/g
#term=$1
$title=Tweet/((*@pageRef/#rule/#term*))/((*@clip*)) 
```

**Reply by arunnbabu81** [reply]
Thank you @buggyj that works
